test/contracts/OVM/accounts/OVM_ECDSAContractAccount.spec.ts

```diff
@@ -227,5 +229,33 @@ describe('OVM_ECDSAContractAccount', () => {
         'Transaction chainId does not match expected OVM chainId.'
       )
     })
+
+    it(`should revert on insufficient gas`, async () => {
+      const alteredInsufficientGasTx = {
+        ...DEFAULT_EIP155_TX,
+        gasLimit : 88888888
+      }
+      const message = serializeNativeTransaction(alteredInsufficientGasTx)
+      const sig = await signNativeTransaction(wallet, alteredInsufficientGasTx)
+
+      await callPrecompile(
```

Comment:
Might be worth adding a custom `gasLimit` field to this test function so that we can explicitly assert the gas failure, not just by setting an implicitly too high of `gasLimit`.